f5lbaasdriver/v2/bigip/plugin_rpc.py

```diff
@@ -812,3 +812,47 @@ def validate_listeners_state(self, context, listeners, host=None):
                               e.message)
                     listener_status[listener_id] = 'Unknown'
         return listener_status
+
+    # validate a list of l7policys id - assure they are not deleted
+    @log_helpers.log_method_call
+    def validate_l7policys_state_by_listener(self, context, listeners,
+                                             host=None):
```

Comment:
It appears that the host parameter isn't used.  Can we remove it?

Reply:
I'm not certain. I simply followed the same scheme that the rest of them did. I will test to assure.